notebooker/execute_notebook.py

```diff
@@ -382,16 +382,17 @@ def docker_compose_entrypoint():
 
     Examples
     --------
-    $ notebooker_execute --report-name watchdog_checks --mongo-host mktdatad
+    $ notebooker_execute --mongo-host mktdatad execute-notebook --report-name watchdog_checks
```

Comment:
Appreciate it was here previously, but perhaps we should use something more generic than `mktdatad` as an example mongo host?

Reply:
Should be `notebooker-cli` instead of `notebooker_execute`